package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "to-do-list-",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --mode production"
```

Comment:
- [ ] **Optional:** The steps to run your code isn't specified and the general start script is missing. Kindly add `"start": "webpack serve --open"` to `package.json` script section.
- [x] **Optional** Kindly update your scripts with the below :
 `"build": "webpack",
    "watch": "webpack --watch",
    "start": "webpack serve --open"`